awq_cuda/quantization/gemm_cuda_gen.cu

```diff
@@ -86,7 +86,7 @@ __global__ void __launch_bounds__(64) gemm_forward_4bit_cuda_m16n128k32(int G, i
                             + (((int)threadIdx.x) % (128 / 8)) * 8;
 
   half* C_ptr = C 
-              + static_cast<long long>(blockIdx_z) * M * OC        // blockIdz.x -> split_k dim
```

Comment:
This cast was added to avoid an overflow. Goes for all places with `blockIdx_z * M * OC`
https://github.com/vllm-project/vllm/pull/1295

Reply:
Thank you. Made replacements in the code